### PR TITLE
Zip logs and wallet db (testnet only) when sending a bug report

### PR DESCRIPTION
**Pull request by Jasonvdb**
## Description

- Zip logs before attaching to bug report mail
- Include wallet DB in bug report if testnet
- Cleanup old zip files after a day
- Improved error message for when users don't have the Apple mail app setup

## Types of changes

* [ ] UI fix (non-breaking change which fixes a UI issue)
* [x] Functionality bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [ ] I have tested this on multiple simulator devices with different screensizes.
* [x] I'm merging against the `development` branch.
* [x] Commits have been squashed into one commit with a descriptive commit message
* [x] I ran all tests before pushing.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added/changed tests to cover my changes if not UI changes.

### CODE REVIEW

File: MobileWallet/Screens/Debug/UIViewControllerDebugExtension.swift [hunk 1]
```diff
@@ -40,6 +40,14 @@
 
 import UIKit
 import MessageUI
+import ZIPFoundation
+
+private enum DebugErrors: Error {
```
Comment (kukabi) [inline MobileWallet/Screens/Debug/UIViewControllerDebugExtension.swift:45]: Singular names are better for type names I think.

File: MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift [hunk 2]
```diff
@@ -71,33 +71,60 @@ private func getUnusedLogFiles() -> [URL] {
...
+            try fileManager.removeItem(at: oldestFile)
+            TariLogger.info("Deleting log file: \(oldestFile.lastPathComponent)")
+        } catch {
+            TariLogger.error("Failed to delete log file", error: error)
+            break
+        }
+    }
+}
+
+private func bugReportZipFilesCleanup() {
```
Comment (kukabi) [inline MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift:103]: I think you can safely delete any zip files at the beginning of a new log mail request (thinking a zip is prepped only for a single request) - this way you'd have max 1 zip file sitting at any time.

Comment (kukabi) [reply MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift:103]: A different story if you'd like to access them later of course.

Comment (Jasonvdb) [reply MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift:103]: Originally that's what I was doing but then I decided if they never report another bug then it's a few mb that stays forever with no purpose

Comment (kukabi) [reply MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift:103]: I see you're right - now I have to do the same thing on Android 😄

Comment (Jasonvdb) [reply MobileWallet/TariLib/Wrappers/Utils/StorageCleanup.swift:103]: Awesome 👍

**Pull request merged**